tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/fullscreen_image.py

```diff
@@ -38,7 +39,7 @@ def is_photo_toolbar_displayed(self):
 
     @property
     def current_image_source(self):
-        return self.marionette.find_element(*self._current_image_locator).value_of_css_property('background-image')
+        return self.marionette.find_element(*self._current_image_locator).get_attribute('src')
```

Comment:
I switched gallery to use <img src=""> instead of using a css background-image, so this fixes the tests to match.